src/library/librarycontrol.cpp

```diff
@@ -482,16 +482,26 @@ void LibraryControl::emitKeyEvent(QKeyEvent&& event) {
     VERIFY_OR_DEBUG_ASSERT(m_pLibraryWidget) {
         return;
     }
-    if (!m_pLibraryWidget->hasFocus() && !m_pSidebarWidget->hasFocus()) {
+    if (!QApplication::focusWindow()) {
+        qDebug()<<"Mixxx window is not focused, don't send key events";
+    }
+
+    Qt::Key key = static_cast<Qt::Key>(event.key());
+//    qDebug() << "|   key  " << key;
+
+    // If the main window has focus, any widget can receive Tab.
+    // Other keys should be sent to library widgets only to not
+    // accidentally alter spinboxes etc.
+    if (key != Qt::Key_Tab && !m_pSidebarWidget->hasFocus()
+            && !m_pLibraryWidget->getActiveView()->hasFocus()) {
         setLibraryFocus();
     }
-    auto focusWidget = QApplication::focusWidget();
-    VERIFY_OR_DEBUG_ASSERT(focusWidget) {
-        return;
+    if (key == Qt::Key_Tab && !QApplication::focusWidget()){
+        setLibraryFocus();
     }
     // Send the event pointer to the currently focused widget
     for (auto i = 0; i < event.count(); ++i) {
```

Comment:
What is this loop for? The send event will include the count. 
In which cases is count set?

Reply:
I was under the impression mulitple `MoveDown` events might be sent at once, but doesn't seem to be possible.
Even though I didn't introduce this I'll double check. Maybe we can remove it. Then my [commment above](https://github.com/mixxxdj/mixxx/pull/2599#discussion_r398891942) doesn't apply.

Reply:
I'll leave this as is.
`slotScrollVertical(double v)` and `slotMoveHorizontal(double v)` for example can send multiple key presses.